src/workerd/api/actor-state.c++

```diff
@@ -778,6 +778,36 @@ kj::Array<jsg::Ref<api::WebSocket>> DurableObjectState::getWebSockets(jsg::Lock&
   return kj::Array<jsg::Ref<api::WebSocket>>();
 }
 
+void DurableObjectState::setWebSocketAutoresponse(kj::String request, kj::String response) {
+  auto maxRequestResponseSize = 2048;
```

Comment:
I thought we agreed on 4k per string? Or was it 4k shared among both @koeninger?

Reply:
current docs PR for attachment size has 2k https://github.com/cloudflare/cloudflare-docs/pull/7872/files#diff-4a15dd9f0af093a3924d21d8eea20d82396bdb2f13c81cf26516e8e328b8543bR311  because that's the limit for a DO storage key.  Starting there for this as well seems fine to me.